Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:10.16.3-alpine AS node
+WORKDIR /app
+
+
+
+#
+# Stage: NPM install
+#
+FROM node AS npm
+
+COPY package.json \
+    package-lock.json \
+    ./
+
+RUN npm install
```

Comment:
Is this the right place to talk about `npm` vs. `yarn`? I noticed that reviewer is using yarn and the pattern library is using npm.